src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Application {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[6];
+
+        for (int i = 0; i < figures.length; i++) {
+            if (i % 2 == 0) {
+                figures[i] = figureSupplier.getDefaultFigure();
+                figures[i].draw();
```

Comment:
this common part should be extracted, for just one line left we can use ternary operator